library/solr

```diff
@@ -1,75 +1,85 @@
-# this file is generated via https://github.com/docker-solr/docker-solr/blob/d6e4780449d3517718e794e59c46214ee3a79f50/generate-stackbrew-library.sh
+# this file is generated via https://github.com/docker-solr/docker-solr/blob/426f7e3ead0c7671556ed8998fb0d345d1616ea5/generate-stackbrew-library.sh
 
 Maintainers: Martijn Koster <mak-github@greenhills.co.uk> (@makuk66),
              Shalin Mangar <shalin@apache.org> (@shalinmangar)
 GitRepo: https://github.com/docker-solr/docker-solr.git
 
-Tags: 8.1.1, 8.1, 8, latest
+Tags: 8.2.0, 8.2, 8, latest
 Architectures: amd64, arm64v8
-GitCommit: 02fff54d99672ee2666bff6f96f008996fb3e3b8
+GitCommit: 677d032a1dc960be7aa725cd9633f06f139a8aa7
```

Comment:
This commit seems a little behind, did you want to also include https://github.com/docker-solr/docker-solr/commit/a89957c6b5242ec4b72e539e39cafe27b7e39a6d?

Reply:
That finished tests <15m ago :-) But yeah, it would be good to have. I'll update that PR in a sec.